### ISSUE TITLE
Go to raining gold lane

### ISSUE THREAD

**Issue by Gerarddp** [mouseas/steamSummerMinigame#15]
https://github.com/mouseas/steamSummerMinigame/issues/8#issuecomment-111571319

**Comment by DarkLordJack** [comment]
Added from downstream in 99151b527cccdbdd4839a1481770f5b2238236a4 and 
2dedb22217a87e8fbe596472f9bc504b829c9d5a.

**Comment by luxzg** [comment]
Yes please +1 on this. At least when a spawner is alive in that lane. Since script is autoclicking now with Gold Rain, this can get you more gold than catching lanes with monsters with low health
